features/roles.feature

```diff
@@ -20,33 +20,44 @@ Feature: Manage WordPress roles
     When I run `wp role reset author`
     Then STDOUT should be:
       """
-      Success: Reset 0/1 roles.
+      No changes necessary for 'author' role.
+      Success: Roles reset 0/1.
       """
 
     When I run `wp cap remove author read`
     And I run `wp role reset author`
     Then STDOUT should be:
       """
-      Success: Reset 1/1 roles.
+      Restored 10 capabilities to and removed 9 capabilities from 'author' role.
```

Comment:
Is this message correct? If only one capability was removed originally, shouldn't the message indicate that 1 capability was restored?